src/plugins/mozprefs/README.md

```diff
@@ -40,28 +40,32 @@ In Mozilla preference files `.` is used to separate sections, while elektra uses
 will all result in `lockPref("a.lock.key", "lock");`
 
 ## Example ##
-
-    % kdb mount prefs.js user/prefs mozprefs
-    % kdb setmeta user/prefs/lock/a/lock/key type boolean
-    % kdb set user/prefs/lock/a/lock/key true
-    % kdb setmeta user/prefs/pref/a/default/key type string
-    % kdb set user/prefs/pref/a/default/key "i'm a default key"
-    % kdb setmeta user/prefs/user/a/user/key type integer
-    % kdb set user/prefs/user/a/user/key 123
-
-    % kdb export user/prefs
-
-    [lock/a/lock]
-    key = true
-    [pref/a/default]
-    key = i'm a default key
-    [user/a/user]
-    key = 123
-
-
-    % cat `kdb file user/prefs`
-
-    lockPref("a.lock.key", true);
-    pref("a.default.key", "i'm a default key");
-    user_pref("a.user.key", 123);
-
+```sh
+# Backup-and-Restor:/examples/examples/prefs
+sudo kdb mount prefs.js /examples/prefs mozprefs
+kdb setmeta user/examples/prefs/lock/a/lock/key type boolean
+kdb set /examples/prefs/lock/a/lock/key true
+kdb setmeta user/examples/prefs/pref/a/default/key type string
+kdb set /examples/prefs/pref/a/default/key "i'm a default key"
+kdb setmeta user/examples/prefs/user/a/user/key type integer
+kdb set /examples/prefs/user/a/user/key 123
+#
+kdb export user/examples/prefs ini
+[lock/a/lock]
+key = true
+[pref/a/default]
+key = i'm a default key
+[user/a/user]
+key = 123
+#
+#
+$ cat `kdb file user/examples/prefs`
```

Comment:
Is this also executed via shell recorder?

Are also redirection and HERE documents supported?